### PR TITLE
build: update base docker images

### PR DESCRIPTION
**Pull request by ChristianMurphy**
##### Checklist

-   [x] the [individual contributor license agreement][] is signed
-   [x] commit message follows [commit guidelines][]
-   [x] tests are included
-   [x] documentation is changed or added
-   [x] [message properties][] have been updated with new phrases
-   [x] view conforms with [WCAG 2.0 AA][]

##### Description of change

Update base docker images to supported JDK and Gradle versions.

[individual contributor license agreement]: https://github.com/Jasig/uPortal/blob/master/.github/CONTRIBUTING.md#individual-contributor-license-agreement
[commit guidelines]: https://github.com/Jasig/uPortal/blob/master/.github/CONTRIBUTING.md#commit
[message properties]: https://github.com/Jasig/uPortal/tree/master/uportal-war/src/main/resources/properties/i18n
[WCAG 2.0 AA]: https://www.w3.org/WAI/WCAG20/quickref/?levels=aaa&technologies=smil%2Cpdf%2Cflash%2Csl

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -3,7 +3,7 @@
 # image.
 
 # This image aims for maximum slimness
-FROM openjdk:8-jdk-alpine
+FROM amazoncorretto:8-alpine-jdk
```
Comment (ChristianMurphy) [inline docker/Dockerfile:6]: alternatively adoptopenjdk or another open source long term support vendor could be used.

Comment (ChristianMurphy) [reply docker/Dockerfile:6]: a related note, the `gradle` image builds on adoptopenjdk

Comment (cbeach47) [reply docker/Dockerfile:6]: What's the concern with `openjdk:8-jdk-alpine`?

Comment (ChristianMurphy) [reply docker/Dockerfile:6]: Oracle openJDK 8 stopped receiving updates two years ago as part of https://www.oracle.com/java/technologies/java-se-support-roadmap.html (https://hub.docker.com/layers/openjdk/library/openjdk/8-jdk-alpine/images/sha256-a3562aa0b991a80cfe8172847c8be6dbf6e46340b759c2b782f8b8be45342717?context=explore was the last release)
which has led to some security warnings https://snyk.io/test/docker/openjdk%3A8-jdk-alpine

Corretto openJDK 8 (https://aws.amazon.com/about-aws/whats-new/2020/08/amazon-corretto-8-11-support-extended/) and AdoptOpenJDK 8 (https://adoptopenjdk.net/support.html?variant=openjdk8&jvmVariant=hotspot#roadmap) are supported until 2026

Comment (cbeach47) [reply docker/Dockerfile:6]: Cool, thx @ChristianMurphy 

@bjagg - do you know of any reasons to choose Corretto vs AdoptOpenJDK for the docker images?

Comment (bjagg) [reply docker/Dockerfile:6]: Nope!

Comment (ChristianMurphy) [reply docker/Dockerfile]: Went for AdoptOpenJDK as the base for both, happy to reconsider if others feels strongly otherwise.

**Review by jonathanmtran — APPROVED** [review]

**Pull request merged**